content/docs/user-guide/gems/reference/environment/vegetation/procedures/selection-random.md

```diff
@@ -6,67 +6,77 @@ title: Creating Weight-Based Random Selection
-**Note**
+    1. In the **Vegetation Asset List** component's properties, next to **Embedded Assets**, click **+**.
+
+        {{< note >}}
 When specifying your first asset, you don't need to click + because there is an empty asset displayed by default. Specify the mesh.
-![\[In the Vegetation Asset List component's properties, next to Embedded Assets, click the plus sign.\]](/images/user-guide/vegetation/dynamic/dynamic-vegetation-procedures-gradient-random-selection-browse.png)
+{{< /note >}}
+
+        ![In the Vegetation Asset List component's properties, next to Embedded Assets, click the plus sign.](/images/user-guide/vegetation/dynamic/dynamic-vegetation-procedures-gradient-random-selection-browse.png)
+
+    1. Under the blank asset, listed as **<asset name>**, next to **Mesh Asset**, click **Browse (…)**.
 
-   1. Under the blank asset, listed as **<asset name>**, next to **Mesh Asset**, click **Browse (…)**.
-![\[Under the blank asset, next to Mesh Asset, click Browse (…).\]](/images/user-guide/vegetation/dynamic/dynamic-vegetation-procedures-gradient-random-selection-add-asset.png)
+       ![Under the blank asset, next to Mesh Asset, click Browse (…).](/images/user-guide/vegetation/dynamic/dynamic-vegetation-procedures-gradient-random-selection-add-asset.png)
 
-   1. In the search bar, enter a search term, such as **flower**, and select a vegetation asset.
-![\[In the search bar, enter a search term, such as flower, and select a vegetation asset.\]](/images/user-guide/vegetation/dynamic/dynamic-vegetation-procedures-gradient-random-selection-select-flower.png)
+    1. In the search bar, enter a search term, such as **flower**, and select a vegetation asset.
 
-   Repeat this step with different assets until you have three or more assets.
+        ![In the search bar, enter a search term, such as flower, and select a vegetation asset.](/images/user-guide/vegetation/dynamic/dynamic-vegetation-procedures-gradient-random-selection-select-flower.png)
 
-   Although you have specified multiple assets, only the first asset that you added appears in the Viewport. The next several procedures show you how to distribute the vegetation so that all of them appear.
+        Repeat this step with different assets until you have three or more assets.
+
+        Although you have specified multiple assets, only the first asset that you added appears in the Viewport. The next several procedures show you how to distribute the vegetation so that all of them appear.
 
 1. Click **Add Component**, **Vegetation Asset Weight Selector**.
 
    In the next step, [Creating a Gradient Entity](#create-gradient-entity), you link this component to the gradient, which supplies values between 0.0 to 1.0 at a given position. Assets are then mapped based on those values and each asset's weight and order values.
 
-## Creating a Gradient Entity 
+## Creating a Gradient Entity {#create-gradient-entity}
```

Comment:
Avoid the use of manually-defined anchors in headers; the autogenerated anchor here is named `creating-a-gradient-entity`.

```suggestion
## Creating a Gradient Entity
```